resources/engines.json

```diff
@@ -69,181 +87,85 @@
     "Gecko_1_9_2": {
       "inherits": "Gecko",
       "properties": {
-        "RenderingEngine_Version": "1.9.2"
+        "RenderingEngine_Version": "1.9.2",
+        "CssVersion": "3"
       }
     },
     "Gecko_1_9_3": {
       "inherits": "Gecko",
       "properties": {
-        "RenderingEngine_Version": "1.9.3"
+        "RenderingEngine_Version": "1.9.3",
+        "CssVersion": "3"
       }
     },
     "Gecko_2": {
       "inherits": "Gecko",
       "properties": {
-        "RenderingEngine_Version": "2.0"
+        "RenderingEngine_Version": "2.0",
+        "CssVersion": "3"
       }
     },
     "Gecko_2_2": {
       "inherits": "Gecko",
       "properties": {
-        "RenderingEngine_Version": "2.2"
-      }
-    },
-    "Gecko_4": {
```

Comment:
I take it that these versions of Gecko aren't directly used anymore (in favor of the Gecko_Dynamic engine).

Reply:
These versions became unused, when different Firefox files were merged.